README.md

```diff
@@ -31,7 +31,7 @@ and [custom functions](https://handsontable.github.io/hyperformula/guide/custom-
 ## Documentation
 
 - [Explainer video](https://www.youtube.com/watch?v=JJXUmACTDdk) /
-[Blog post](https://handsontable.com/blog/articles/2020/6/introducing-hyperformula-fast-javascript-calculation-engine)
+[Blog post](https://handsontable.com/blog/articles/2020/7/introducing-hyperformula-fast-javascript-calculation-engine)
```

Comment:
no longer a relevant change, right @krzysztofspilka ?

Reply:
I can confirm that. 

The link https://handsontable.com/blog/articles/2020/7/introducing-hyperformula-fast-javascript-calculation-engine does not exist!

We should keep the old one: https://handsontable.com/blog/articles/2020/6/introducing-hyperformula-fast-javascript-calculation-engine